core/domain/exp_domain.py

```diff
@@ -2460,6 +2513,48 @@ def get_all_html_content_strings(self):
 
         return html_list
 
+    @classmethod
+    def to_proto(
+            cls, exploration_id, title, version, init_state_name, states):
+        """Calculate the exploration size by setting exploration proto.
+
+        Args:
+            exploration_id: str. The id of the exploration.
+            title: str. The exploration title.
+            version: int. The version of the exploration.
+            init_state_name: str|None. The name of the initial state.
+            states: list|None. The list of state
+                domain objects.
+
+        Returns:
+            Proto Object. The exploration proto object.
+        """
+        state_protos = {}
+        exploration_proto = {}
+        if states is not None:
+            state_protos = state_domain.State.to_proto(states)
+
+        if isinstance(title, str):
+            exploration_proto = exploration_pb2.Exploration(
+                id=exploration_id,
+                content_version=version,
+                init_state_name=init_state_name,
+                title=title,
+                states=state_protos)
+        return exploration_proto
+
+    @classmethod
+    def calculate_size_of_proto(cls, proto):
+        """Calculate the byte size of the proto object.
+
+        Args:
+            proto: proto object. The proto object.
+
+        Returns:
+            Int. The byte size of the proto object.
```

Comment:
typeinfo in docstrings should be whatever you get when you do `type(.)` on the return value. In this case, I'm not sure if "Int" should be capitalized.

Reply:
This one is `int` only. 

`<class 'int'>`

Reply:
Done